lib/utils/keyCommon.js

```diff
@@ -17,21 +17,32 @@ export default class KeyCommon {
   /**
    * @return  {Object} Returns an Key Pair Object with Private and Public keys
    */
-  static createKeyPair() {
-    const privateKeyBytes = secureRandom.randomUint8Array(32);
+  static createKeyPair({rawPrivateKey = ''} = {}) {
+    let privateKeyBytes;
+
+    if (rawPrivateKey) {
+      if (typeof rawPrivateKey === 'string' && rawPrivateKey.length === 32) {
+          privateKeyBytes = rawPrivateKey;
```

Comment:
This is actually problematic, it can result in invalid private keys being generated. The fix is super simple, though. Just replace the line here (`privateKeyBytes = rawPrivateKey`) with the following instead:

          privateKeyBytes = Uint8Array.from(rawPrivateKey);

Reply:
Fixed!